javagen/src/main/java/com/azure/autorest/implementation/ClientModelPropertiesManager.java

```diff
@@ -52,6 +52,7 @@ public final class ClientModelPropertiesManager {
     private final List<ClientModelProperty> superRequiredProperties;
     private final List<ClientModelProperty> superSetterProperties;
     private final List<ClientModelProperty> superReadOnlyProperties;
+    private final ClientModelProperty superAdditionalProperties;
```

Comment:
Should we call it `superAdditionalPropertiesProperty`?

Currently follow naming of `private final ClientModelProperty additionalProperties;`